discourse/supervisord-append.conf

```diff
@@ -1,8 +1,8 @@
 [program:postgres]
 user = user
-command = /usr/lib/postgresql/9.4/bin/postgres
+command = sudo -u postgres /usr/lib/postgresql/9.5/bin/postgres -D /etc/postgresql/9.5/main/
```

Comment:
Nit: Instead of using `sudo -u` in the `command`, I think you could simply change `user = user` to `user = postgres` on the line above.

Reply:
Not unless supervisord is running as root.